examples/blink-led-morse.ms

```diff
@@ -0,0 +1,39 @@
+;; @ardumont
+;; microscheme.org
+
+(include "src/stdlib.ms")
```

Comment:
Hi Antoine. The standard library is actually loaded automatically, so there's no need to include it. (The list.ms, vector.ms and io.ms libraries are now redundant.) Thanks for the contribution...